test/1_test_fullpath.js

```diff
@@ -1361,18 +1359,16 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
       const transaction = await web3.eth.getTransaction(txRefund.tx);
       const transactionBlock = await web3.eth.getBlock(transaction.blockNumber);
-      assert.isTrue(
-        voucherStatus.complainPeriodStart.eq(new BN(transactionBlock.timestamp))
-      );
+      assert.isTrue(voucherStatus[3].eq(new BN(transactionBlock.timestamp)));
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)